addons/unattended-upgrades/README.md

```diff
@@ -0,0 +1,41 @@
+# Unattended Upgrades
+
+This addon will automate upgrading system packages of the distro of your choice.
+
+## Requirements
+
+Since KubeOne 1.3+ we automatically label control-plane nodes with
+`v1.kubeone.io/operating-system` and worker nodes with
+`v1.machine-controller.kubermatic.io/operating-system` and use those labels as
+nodeAffinity in this addon manifests.
+
+## What's included
+
+This addon provides bunch of DaemonSets and operators:
+
+* **Debian/Ubuntu**
+  DaemonSet that will install `unattended-upgrades`
+* **RHEL/CentOS**
+  DaemonSet that will install and configure `yum-cron`/`dnf-automatic`
+* **Debian/Ubuntu/RHEL/CentOS**
```

Comment:
Do we also want to deploy `kured` on Amazon Linux 2?

Reply:
yeah, done